config/samples/numaplane.numaproj.io_v1alpha1_pipelinerollout.yaml

```diff
@@ -16,7 +16,7 @@ spec:
           source:
             # A self data generating source
             generator:
-              rpu: 5
+              rpu: 500
```

Comment:
higher RPU can be better for testing PPND and seeing whether the pausing occurred at the right time, since it takes longer to pause